third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py

```diff
@@ -36,15 +36,19 @@ def compare_enums(autogen_context, upgrade_ops, schema_names):
 			metadatas = autogen_context.metadata
 		enums = get_declared_enums(metadatas, schema, default)
 		for table, values in enums.items():
-			if is_table_present(table, autogen_context.connection):
-				items = {r[0] for r in autogen_context.connection.execute("SELECT item_id FROM {}".format(table))}
+			fully_qualified_table_name = "%s.%s" % (schema, table) if schema else table
+			if is_table_present(fully_qualified_table_name, autogen_context.connection):
+				items = {
+					r[0]
+					for r in autogen_context.connection.execute(
+						"SELECT item_id FROM {}".format(fully_qualified_table_name)
+					)
```

Comment:
should we use bind parameters for this? 

`sql = sql.text("SELECT item_id FROM :tablename")`
`autogen_context.connection.execute(sql, tablename=fully_qualified_table_name)`

Reply:
I'll fix this in #59, as this code actually get yoinked there.

Reply:
ok makes sense